### DISCUSSION TITLE
De length in console.app and the xml is continual changing.

### DISCUSSION THREAD

**Discussion by BouBal** [sparkle-project/Sparkle#2679] [Q&A]
The console gives continuous another length, but local and on the server side the are exactly the same. I cannot found other problem.. Also I have tried .dmg and zip. both has the same problem. 

Is there somebody that has the samen problem and maybe a fix?

**Comment by zorgiepoo** [comment]
Pleases elaborate and paste the XML and console logs you see. It could be a server caching issue but this report is too vague.
